glide.yaml

```diff
@@ -8,73 +8,73 @@ import:
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/code-generator
   repo:    https://github.com/openshift/kubernetes-code-generator.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/component-base
   repo:    https://github.com/openshift/kubernetes-component-base.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/csi-translation-lib
   repo:    https://github.com/openshift/kubernetes-csi-translation-lib.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-aggregator
   repo:    https://github.com/openshift/kubernetes-kube-aggregator.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-controller-manager
   repo:    https://github.com/openshift/kubernetes-kube-controller-manager.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kubectl
   repo:    https://github.com/openshift/kubernetes-kubectl.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kubelet
   repo:    https://github.com/openshift/kubernetes-kubelet.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-proxy
   repo:    https://github.com/openshift/kubernetes-kube-proxy.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-scheduler
   repo:    https://github.com/openshift/kubernetes-kube-scheduler.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/legacy-cloud-providers
   repo:    https://github.com/openshift/kubernetes-legacy-cloud-providers.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/metrics
   repo:    https://github.com/openshift/kubernetes-metrics.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-apiserver
   repo:    https://github.com/openshift/kubernetes-sample-apiserver.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-cli-plugin
   repo:    https://github.com/openshift/kubernetes-sample-cli-plugin.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-controller
   repo:    https://github.com/openshift/kubernetes-sample-controller.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kubernetes
-  repo:    https://github.com/openshift/kubernetes.git
-  version: origin-4.4-kubernetes-1.16.2
+  repo:    https://github.com/p0lyn0mial/kubernetes.git
```

Comment:
That shouldn't be here, https://github.com/openshift/kubernetes/pull/102 already merged.

Reply:
fixed!, thx.